ftw/contacts/configure.zcml

```diff
@@ -0,0 +1,80 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:plone="http://namespaces.plone.org/plone"
+    xmlns:five="http://namespaces.zope.org/five"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
+    xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    xmlns:profilehook="http://namespaces.zope.org/profilehook"
+    i18n_domain="ftw.contacts">
+
+    <include file="permissions.zcml" />
+    <include package=".browser" />
+    <include file="lawgiver.zcml" zcml:condition="installed ftw.lawgiver" />
+
+    <i18n:registerTranslations directory="locales" />
+
+    <genericsetup:registerProfile
+        name="default"
+        title="ftw.contacts"
+        directory="profiles/default"
+        description="Installs the ftw.contacts package"
+        provides="Products.GenericSetup.interfaces.EXTENSION"
+        />
+    <profilehook:hook
+      profile="ftw.contacts:default"
+      handler=".hooks.default_profile_installed"
+      />
+    <browser:resourceDirectory
+        name="ftw.contacts.resources"
+        directory="resources"
+        />
+    <adapter
+        factory=".adapters.MemberAccessor"
+        for="ftw.contacts.contents.member.IMember"
+        provides="ftw.contacts.interfaces.IMemberAccessor"
+        />
+    <!-- ftw.geo -->
+    <adapter
+        for="ftw.contacts.contents.contact.IContact"
+        provides="collective.geo.settings.interfaces.IGeoCustomFeatureStyle"
+        factory="collective.geo.contentlocations.geostylemanager.GeoStyleManager"
+    />
+    <subscriber
+        for="ftw.contacts.contents.contact.IContact
+             zope.lifecycleevent.interfaces.IObjectModifiedEvent"
+        handler=".handlers.initializeCustomFeatureStyles"
```

Comment:
Please rename `handlers.py` to `subscribers.py`.
Handlers is too ambiguous, it could contain event handlers, setup handlers or any other fancy kind of handler. Subscribers is more accurate and the current best practice.